ProcessMaker/Http/Controllers/Api/TemplateController.php

```diff
@@ -215,13 +215,6 @@ private function checkIfAssetsExist($request)
         $existingOptions = [];
 
         foreach ($payload['export'] as $key => $asset) {
-            if (Str::contains($asset['name'], 'Screen Interstitial')
-                || Str::contains($asset['model'], 'CommentConfiguration')
```

Comment:
Hi @estebangallego, I have a question about this commit: https://github.com/ProcessMaker/processmaker/pull/5578/commits/a126e51eeff3619894f43e783a05f82a5587fbaa related to the ticket [FOUR-10936](https://processmaker.atlassian.net/browse/FOUR-10936). I wonder if it's necessary to remove the line `Str::contains($asset['model'], 'CommentConfiguration')` to prevent a regression of the ticket FOUR-10936. What's do you think?

Reply:
Good call and thank you for noticing, I'll add the CommentConfiguration back!